Orange/widgets/data/owtranspose.py

```diff
@@ -54,9 +57,7 @@ def __init__(self):
         edit.editingFinished.connect(self._apply_editing)
 
         self.meta_button = gui.appendRadioButton(box, "From meta attribute:")
-        self.feature_model = DomainModel(
-            order=DomainModel.METAS, valid_types=StringVariable,
-            alphabetical=True)
+        self.feature_model = DomainModel(DomainModel.METAS, alphabetical=True)
```

Comment:
I don't think it needs to be a meta.

I am thinking about prohibiting discrete variables, though. They are always a bad choice, I suppose.